src/org/labkey/test/components/glassLibrary/grids/GridBar.java

```diff
@@ -30,9 +30,9 @@
     final private WebDriver _driver;
     final private ResponsiveGrid _responsiveGrid;
 
-    protected GridBar(WebElement buttonBar, ResponsiveGrid responsiveGrid, WebDriver driver)
+    protected GridBar(WebDriver driver, WebElement container, ResponsiveGrid responsiveGrid)
     {
-        _gridBarElement = buttonBar;
+        _gridBarElement = Locator.tagWithClass("div", "query-grid-bar").findElement(container);
```

Comment:
This Locator is defined below. Also, use `findWhenNeeded` so that this component can be initialized lazily:
```suggestion
        _gridBarElement = Locators.gridBar().findWhenNeeded(container);
```

Reply:
Updated.